apps/balancer-tools/src/app/apr/round/(components)/PoolListTable.tsx

```diff
@@ -178,21 +198,53 @@ function TableRow({
   const router = useRouter();
   return (
     <Table.BodyRow
-      classNames="hover:bg-blue4 hover:cursor-pointer"
+      classNames="hover:bg-blue4 hover:cursor-pointer duration-500"
       onClick={() => {
         router.push(poolRedirectURL);
       }}
     >
-      <Table.BodyCell>
-        {symbol} ({networkFor(network)})
-      </Table.BodyCell>
-      <Table.BodyCell padding="py-4 px-1">{formatNumber(tvl)}</Table.BodyCell>
-      <Table.BodyCell padding="py-4 px-1">
+      <Table.BodyCellLink href={poolRedirectURL} tdClassNames="w-6">
+        <Image
+          src={`/assets/network/${networkFor(network)}.svg`}
+          height={25}
+          width={25}
+          alt={`Logo for ${networkFor(network)}`}
+        />
+      </Table.BodyCellLink>
+      <Table.BodyCellLink
+        href={poolRedirectURL}
+        linkClassNames="gap-2"
+        tdClassNames="w-11/12"
+      >
+        {tokens.map((token) => (
+          <div className="relative mx-1 flex max-h-10 items-center rounded-md px-2 py-1 bg-blue6">
+            {token.symbol}
+            {token.weight ? (
+              <span className="text-xs ml-1 text-slate-400">
+                {(parseFloat(token.weight) * 100).toFixed(2)}%
+              </span>
+            ) : (
+              ""
+            )}
+          </div>
+        ))}
+      </Table.BodyCellLink>
+
+      <Table.BodyCellLink href={poolRedirectURL}>
+        {PoolTypeEnum[poolType]}
+      </Table.BodyCellLink>
+
+      <Table.BodyCellLink href={poolRedirectURL}>
+        {formatNumber(tvl)}
```

Comment:
Think we should call `tvl.toLocaleString("en-US")` here to preserve the actual values like displayed in Balancer's UI 🤔 wdyt?

Reply:
Makes sense, another thing that theyrs look better is to not show decimal numbers

Reply:
Right -- but then for votes this is mandatory IMO since many will have sub0 voting shares.